src/seam-connect/client.ts

```diff
@@ -73,8 +80,8 @@ export class Seam extends Routes {
       baseURL: endpoint,
       headers: {
         ...axiosOptions?.headers,
-        Authorization: `Bearer ${apiKey}`,
-        ["User-Agent"]: `Javascript SDK v${version} (https://github.com/seamapi/javascript)`,
+        Authorization: `Bearer ${apiKey || clientAccessToken}`,
+        // ["User-Agent"]: `Javascript SDK v${version} (https://github.com/seamapi/javascript)`,
```

Comment:
```suggestion
        ["User-Agent"]: `Javascript SDK v${version} (https://github.com/seamapi/javascript)`,
```

Reply:
What are the errors? I'm assuming the browser doesn't like when JavaScript tries to override the user agent header. If so we probably need to selectively disable this in the browser environment.

Reply:
yeah we use the user agent in general to track SDK usage

maybe we could do a `typeof window === "undefined"` check?

Reply:
maybe we could 
1. keep `user-agent` for now until tooling is updated
2. disable `user-agent` when in a browser
3. always send a second header like `x-seam-sdk-version` with the same info